Python/bytecodes.c

```diff
@@ -4072,6 +4072,11 @@ dummy_func(
             TIER_TWO_ONLY
             value = ptr;
         }
+        pure op (_POP_TOP_LOAD_CONST_INLINE_BORROW, (ptr/4, pop -- value)) {
+            TIER_TWO_ONLY;
+            DECREF_INPUTS();
```

Comment:
Since there is only one input
```suggestion
            Py_DECREF(pop);
```